Common: minimosd gets serial-protocol note (#4231)
ArduPilot/ardupilot_wiki: common/source/docs/common-minim-osd-quick-installation-guide.rst

The second stage regulator from the MinimOSD boards earlier than V1.1

> gets too hot on 12V video setups. If your frame has not a good air flow

Reviewer: ```suggestion
gets too hot on 12V video setups. If your frame does not have good air flow
```

for cooling the OSD board you may want to feed the OSD entirely from
the autopilot. Probably it will add some noises from servos, but you'll be more

> safe by this way:

Reviewer: ```suggestion
safe this way:
```

Author: existing issues but fixed, txs
